### PR TITLE
(ci): include .md file changes to trigger website validation

### PR DESCRIPTION
**Pull request by zhenmichael**
Updating CI to run website validation for changes to any .md files instead of just changes to the "docs/" directory. 

Currently, `markdown-magic` generates some of the content in the docs using content from sources outside the docs directory including package READMEs throughout the repo. We want to validate these changes as they could implicitly break the website content.

### CODE REVIEW

File: .github/workflows/website-validation.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ on:
     types: [opened, synchronize, reopened]
     paths:
       - "docs/**"
+      - "*.md"
+      - "**/*.md"
```
Comment (tylerbutler) [inline .github/workflows/website-validation.yml:8]: Is it possible to exclude, say, the server and build-tools folders?

Comment (tylerbutler) [reply .github/workflows/website-validation.yml:8]: If it's not straightforward then ignore it.

Comment (zhenmichael) [reply .github/workflows/website-validation.yml:8]: Updated to exclude the folders using `paths-ignore`

File: .github/workflows/website-validation.yml [hunk 2]
```diff
@@ -4,6 +4,11 @@ on:
...
+    paths-ignore:
+      - "build-tools/**"
+      - "server/**"
```
Comment (tylerbutler) [inline .github/workflows/website-validation.yml:11]: Are you sure this works? From [these docs](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore):

> You cannot use both the paths and paths-ignore filters for the same event in a workflow. If you want to both include and exclude path patterns for a single event, use the paths filter prefixed with the ! character to indicate which paths should be excluded.

Comment (zhenmichael) [reply .github/workflows/website-validation.yml]: Updated the path exclusion. Thanks for the correction!

**Review by tylerbutler — APPROVED** [review]

**Pull request merged**